stackanetes/common/utils.py

```diff
@@ -21,3 +25,35 @@ def env(*args, **kwargs):
         if value:
             return value
     return kwargs.get('default', '')
+
+
+def get_kubectl_command(add_namespace=True):
+    cmd = [CONF.stackanetes.kubectl_path]
+    if CONF.stackanetes.host:
+        server = "--server=" + CONF.stackanetes.host
+        cmd.append(server)
+    if CONF.stackanetes.kubeconfig_path:
+        kubeconfig_path = "--kubeconfig=" + CONF.stackanetes.kubeconfig_path
+        cmd.append(kubeconfig_path)
+    if CONF.stackanetes.context:
+        context = "--context=" + CONF.stackanetes.context
+        cmd.append(context)
+    if add_namespace:
```

Comment:
What about namespace conf parameter check?
`if add_namespace and CONF.stackanetes.namespace`